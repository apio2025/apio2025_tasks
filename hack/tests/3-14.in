gcxAq5ZVsD9pstWtmGwYDYtXCn4B88dd
10 1
834703043
578986637
651502762
414650237
443519800
705351284
38392338
693410274
773944380
577662841
