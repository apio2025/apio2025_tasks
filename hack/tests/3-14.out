1eJ4ArSX1VUyR4oqcU3vVcJ6xoN8jIYs
1.000000
All good
